freqtrade/analyze.py

```diff
@@ -25,11 +25,24 @@ class SignalType(Enum):
     SELL = "sell"
 
 
+class SellType(Enum):
+    """
+    Enum to distinguish between sell reasons
+    """
+    ROI = "roi"
+    STOP_LOSS = "stop_loss"
+    TRAILING_STOP_LOSS = "trailing_stop_loss"
+    SELL_SIGNAL = "sell_signal"
+    FORCE_SELL = "force_sell"
+    NONE = ""
```

Comment:
I think we can drop `SeelType.NONE` and stick to the built-in `None` as the sqlalchemy column is also nullable

Reply:
I had that initially - but mypy is a bitch with "optional" arguments, and requires to always check for None (even if it's obvious to me, as "none" is only used with "false" ... )

Reply:
Alright then